bucket/peazip.json

```
@@ -0,0 +1,59 @@
+{
+    "version": "6.8.1",
+    "description": "A free file archiver utility, based on Open Source technologies",
+    "homepage": "https://www.peazip.org/peazip-portable.html",
+    "license": {
+        "identifier": "LGPL-3.0-only",
+        "url": "https://www.peazip.org/peazip-tos-privacy.html#software_copyright"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://osdn.net/frs/redir.php?m=dotsrc&f=peazip/71229/peazip_portable-6.8.1.WIN64.zip#/dl.zip",
+            "hash": "098feb0eb36bec848fd36c0d7712815d2b7ac7ddafb9f62642cec10a42e5002a",
+            "extract_dir": "peazip_portable-6.8.1.WIN64"
+        },
+        "32bit": {
+            "url": "https://osdn.net/frs/redir.php?m=dotsrc&f=peazip/71229/peazip_portable-6.8.1.WINDOWS.zip#/dl.zip",
+            "hash": "5e2c7e5820af9ce2b538461d8c0ccf636056f94a6e6fa0b740d355d91f0c79d9",
+            "extract_dir": "peazip_portable-6.8.1.WINDOWS"
+        }
+    },
+    "persist": [
+        "res\\altconf.txt",
+        "res\\bookmarks.txt",
+        "res\\conf.txt",
+        "res\\conf-lastgood.txt",
+        "res\\custedit.txt"
+    ],
+    "bin": "peazip.exe",
+    "shortcuts": [
+        [
+            "peazip.exe",
+            "PeaZip"
+        ]
+    ],
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\res\\bookmarks.txt\")) { New-Item \"$dir\\res\\bookmarks.txt\" | Out-Null }",
+        "if(!(Test-Path \"$persist_dir\\res\\custedit.txt\")) { New-Item \"$dir\\res\\custedit.txt\" | Out-Null }"
+    ],
+    "checkver": {
+        "url": "https://osdn.net/projects/peazip/",
+        "regex": "<a href=\"/projects/peazip/releases/(?<release>[\\d]*)\">peazip ([\\d.]+)</a>"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://osdn.net/frs/redir.php?m=dotsrc&f=peazip/$matchRelease/peazip_portable-$version.WIN64.zip#/dl.zip",
```

Use direct URLs instead of redirector

```suggestion
                "url": "https://dotsrc.dl.osdn.net/osdn/peazip/$matchRelease/peazip_portable-$version.WIN64.zip",
```